src/Hintman.hs

```diff
@@ -44,8 +43,6 @@ runOn cli = do
     let siteString = "https://localhost:" <> show port
     putTextLn ("Starting hintman site at: " <> siteString)
 
-    envConfig <- loadFileConfig "hintman-config.toml"
```

Comment:
Could you also delete `hintman-config.toml` itself?

Reply:
Turns out there's no such file :man_shrugging:

Reply:
:thinking: